src/cli/cli_network_data.cpp

```diff
@@ -244,72 +244,100 @@ template <> otError NetworkData::Process<Cmd("publish")>(Arg aArgs[])
         /**
          * @cli netdata publish dnssrp anycast
          * @code
-         * netdata publish dnssrp anycast 1
+         * netdata publish dnssrp anycast 1 1
          * Done
          * @endcode
-         * @cparam netdata publish dnssrp anycast @ca{seq-num}
+         * @cparam netdata publish dnssrp anycast @ca{seq-num} [@ca{version}]
          * @par
-         * Publishes a DNS/SRP Service Anycast Address with a sequence number. Any current
+         * Publishes a DNS/SRP Service Anycast Address with a sequence number and version. Any current
```

Comment:
Since version is marked as optional, what's the use case for not including it?  The docs you have here imply that it's required.

Reply:
This is mainly for backward compatibility. For example, we want existing scripts using the `cli` command `netdata publish dnssrp anycast 1` without providing a version to continue to work.